libraries/AP_Motors/AP_MotorsHeli_Single.cpp

```diff
@@ -231,8 +231,9 @@ void AP_MotorsHeli_Single::calculate_armed_scalars()
     } else if (_rsc_mode == ROTOR_CONTROL_MODE_CLOSED_LOOP_POWER_OUTPUT) {
         _main_rotor.set_throttle_curve(_rsc_thrcrv.get_thrcrv());
         _main_rotor.set_governor_disengage(_rsc_gov.get_disengage()*0.01f);
-        _main_rotor.set_governor_droop_setting(_rsc_gov.get_droop_setting()*0.01f);
-        _main_rotor.set_governor_setpoint(_rsc_gov.get_setpoint());
+        _main_rotor.set_governor_droop_response(_rsc_gov.get_droop_response()*0.01f);
+        _main_rotor.set_governor_reference(_rsc_gov.get_reference());
```

Comment:
"reference" is a new word.. which is OK but normally we use "target".  I'm actually not a fan of the word "setpoint" but "reference" isn't too clear either I think.  It's up to you guys but I think keeping terminology consistent helps developers move more easily between different parts of the code.

Reply:
This seems to be a duplicate comment on the use of the 'reference' in a droop governor. I would prefer to keep it as designed, as it is not the same as a "target".